packages/contracts/types/hardhat.d.ts

```diff
@@ -1,12 +1,17 @@
 import {BigNumberish, BytesLike} from 'ethers';
 
 export type AragonPluginRepos = {
-  'address-list-voting': string;
-  'token-voting': string;
-  // prettier-ignore
-  'admin': string;
-  // prettier-ignore
-  'multisig': string;
+  AddresslistVotingRepoProxy: string;
+  TokenVotingRepoProxy: string;
+  AdminRepoProxy: string;
+  MultisigRepoProxy: string;
+  // DEPRECATED KEYS
+  // 'address-list-voting': string;
+  // 'token-voting': string;
+  // // prettier-ignore
+  // 'admin': string;
+  // // prettier-ignore
+  // 'multisig': string;
```

Comment:
If the deploy script doesn't use these deprecated keys anymore, we can remove them now already.